### DISCUSSION TITLE
SDXL branch ?

### DISCUSSION THREAD

**Discussion by gitaiuser20** [Linaqruf/kohya-trainer#247] [General]
i read that there's an SDXL branch for kohya, I'm wondering if there are plans to make a colab for it ?

thanks for your great work, your colabs are saviours

**Comment by Linaqruf** [comment]
It's been here for 2 weeks
- https://github.com/Linaqruf/kohya-trainer/blob/main/kohya-trainer-XL.ipynb
- https://github.com/Linaqruf/kohya-trainer/blob/main/kohya-LoRA-trainer-XL.ipynb

However none of them can be used in colab free tier, you need A100 for Kohya Trainer XL and T4 with High Ram for Kohya LoRA Trainer

**Reply by Eden-Luminar** [reply]
It is hard to use XL Collabs at right setting for Free Tier. Any Solution to Free Tier?
